evalml/data_checks/id_columns_data_check.py

```diff
@@ -157,27 +186,50 @@ def validate(self, X, y=None):
         id_cols_above_threshold = {
             key: value for key, value in id_cols.items() if value >= self.id_threshold
         }
+
+        first_col_id = False
+
+        if col_names and col_names[0] in id_cols_above_threshold:
+            first_col_id = True
+
         if id_cols_above_threshold:
-            warning_msg = "Columns {} are {}% or more likely to be an ID column"
+            warning_msg = ""
+            message_code = None
+            action_code = None
+            if first_col_id:
+                warning_msg = (
+                    "The first column '{}' is {}% or more likely to be an ID column"
+                )
+                warning_msg = warning_msg.format(
+                    col_names[0],
+                    self.id_threshold * 100,
+                )
+                message_code = DataCheckMessageCode.HAS_ID_FIRST_COLUMN
+                action_code = DataCheckActionCode.SET_FIRST_COL_ID
+                details = {"column": col_names[0]}
+            else:
```

Comment:
Are we trying to make this an either/or scenario? I think we would still want to know if other columns could be ID columns right?